budgetportal/openspending.py

```diff
@@ -230,6 +230,20 @@ def get_adjustment_kind_ref(self):
         return self.get_ref(self.get_adjustment_kind_dimension(), "label")
 
 
+class BudgetedAndActualExpenditure(BabbageFiscalDataset):
```

Comment:
A different dataset is known as "budgeted and actual expenditure" so we need to rename this

```suggestion
class InYearExpenditure(BabbageFiscalDataset):
```

Reply:
done